public/css/main.css

```diff
@@ -38,4 +38,11 @@ h1 {
 
 .u-bgc--f4f4f4 {
   background-color: #f4f4f4;
-}
\ No newline at end of file
+}
+
+.social-buttons {
+  display: flex;
+  width: 180px;
+  margin: auto;
+  justify-content: space-around;
+}
```

Comment:
Rather than add a new class, can we use tachyons utility classes? http://tachyons.io/